src/patterns/templates/card-layout/card-layout.hbs

```diff
@@ -16,8 +16,6 @@ tips: |
     - `mzp-l-card-quarter`
   - These CSS classes can be applied to any suitable parent element e.g. `<main>`, `<section>`, `<div>`.
   - The grid layout is reversed in right to left (RTL) languages.
-labels:
-    - inprogress
```

Comment:
I thought I'd remove most of these inprogress/prototype labels, as many of these things are now in production